core/common/boot/src/main/java/org/eclipse/edc/boot/system/runtime/BaseRuntime.java

```diff
@@ -85,6 +88,10 @@ public static void main(String[] args) {
     public void boot(boolean addShutdownHook) {
         monitor = createMonitor();
         var config = configurationLoader.loadConfiguration(monitor);
+        if (monitor instanceof ConsoleMonitor) {
```

Comment:
this is not a very clean solution, because the monitor is already used during configuration loading, which could potentially result in log lines at a level that was "configured away". 
BTW this is the reason we put the color option as command line arg, I think the loglevel should be treated the same.